src/fake-timers-src.js

```diff
@@ -1672,7 +1685,7 @@ function withGlobal(_global) {
                                 : NOOP;
                     }
                 });
-            } else if (config.toFake.includes("performance")) {
+            } else if ((config.toFake || []).includes("performance")) {
```

Comment:
seems unrelated? when is it missing, and how hasn't that been an issue before?

Reply:
This has been an issue with Node 16 causing sinon to crash. It's here now in case something in the future reaches this branch

Reply:
> how hasn't that been an issue before?

It was in the latest merged PR #400 and it works on Node 14-15 but not on Node 16  (it was my mistake)

see https://github.com/sinonjs/fake-timers/issues/394#issuecomment-1013758365

Reply:
cool. can we add node 16 (and 17) to CI with this PR?

Reply:
Sure, added Node 16

Reply:
Let me remove 10 which is EoL and has stopped receiving security updates and add Node 17 and a test for something that exists there

Reply:
Node 17 has unrelated failures, let me try and fix those

Reply:
I'll also add a bunch of todos for stuff that doesn't work atm

Reply:
Not totally surehow to get that coverage thing going.

Reply:
removing 10 is a breaking change, fwiw

Reply:
@SimenB Nothing is broken and what we actually run the tests on is not our compatibility test. We have an official policy for all Sinon projects: https://github.com/sinonjs/sinon/blob/master/COMPATIBILITY.md#runtimes which basically says "All supported LTS versions". Once 10 no longer is supported, then it's off the table. No need to bump major version numbers. Not that it matters all that much :smiley:

Reply:
Right, I just saw "Let me remove 10 which is EoL". If that doesn't happen then my comment can be ignored 👍 

---

It's still a breaking change to remove remove support for a runtime, which warrants a new major version regardless of what some markdown file says (unless it also says "we do not follow semver, and a new major means incompatible API change" or some such).

Anyways, this probably got a bit theoretical and not related to the PR at hand 😀

Reply:
> (I don't think supporting EoL versions of Node makes sense since they have bugs/security issues that are never going to be fixed)

I don't disagree with that statement. I'm not saying `@sinonjs/fake-timers` should keep supporting EOL versions of Node indefinitely (or even that this PR shouldn't remove support for it, although that seems orthogonal to fixing support for the current LTS release), but _if_ (when) support is dropped, that's a breaking change which should trigger a new major version.